tests/Decred/TWAnySignerTests.cpp

```diff
@@ -77,8 +77,8 @@ TEST(TWAnySignerDecred, Plan) {
 
     EXPECT_EQ(plan.amount(), 10000000);
     EXPECT_EQ(plan.available_amount(), 39900000);
-    EXPECT_EQ(plan.fee(), 254);
-    EXPECT_EQ(plan.change(), 29899746);
+    EXPECT_EQ(plan.fee(), 225);
```

Comment:
how does it affect Decred?

Reply:
no segwit for decred?

Reply:
Decred has some issues -- let's handle that in separate PR.
- For planning, Decred reuses Bitcoin code, but for signing it duplicates it
- BTC planning code now uses BTC signing for fee computation
- BTC signing and Decred signing differ in keyHash hash function used
- BTC signing fails with key not found
- now missing key case is handled properly in estimation, fee is more accurate
- Decred signing works, but as fee is different, tx will be slightly different
The issue is that Decred tx estimation uses BTC signing, which is incompatible.  Let's address this in different PR; also check if redundancy between Decred&BTC code can be reduced.

<img width="1178" alt="image" src="https://user-images.githubusercontent.com/13562139/84830200-5128d680-b029-11ea-8082-24fb7301386d.png">

Reply:
yes, we can change that later but make sure the fee is enough to broadcast, some chain has  min relay fee, don't want to break it

Reply:
Tested with iOS app, OK
https://github.com/trustwallet/ios/pull/2574#issuecomment-645330107